### PR TITLE
LevelTab & Colours

### PR DESCRIPTION
**Pull request by AltenGD**
with a bg

### CODE REVIEW

File: GDEdit/GDE.App/Main/Colours/GDEColours.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using osuTK.Graphics;
+
+namespace GDE.App.Main.Colours
+{
+    public class GDEColours
+    {
+        /// <summary>
+        /// Returns a Color4 value from a Hex strinhgdefk;joiu 67n5jiyrt;l'fp
```
Comment (AlFasGD) [inline GDEdit/GDE.App/Main/Colours/GDEColours.cs]: What the fuck is that lmao

File: GDEdit/GDE.App/Main/Screens/MainScreen.cs [hunk 2]
```diff
@@ -21,23 +20,17 @@ private void load(TextureStore texStore)
...
-                        Anchor = Anchor.Centre,
-                        Origin = Anchor.Centre
-                    },
-                    new SpriteText
-                    {
-                        Text = ObjectLists.OrbList[0].ToString(),
                         RelativeSizeAxes = Axes.Both,
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Colour = GDEColours.FromHex(@"151515")
```
Comment (AlFasGD) [inline GDEdit/GDE.App/Main/Screens/MainScreen.cs]: Let's not use verbatim (@) strings for colors, please

File: GDEdit/GDE.App/Main/Colours/GDEColours.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        /// </summary>
+        /// <param name="hex">Hex value (pretty self explanitory)</param>
```
Comment (AlFasGD) [inline GDEdit/GDE.App/Main/Colours/GDEColours.cs]: "explanatory"

File: GDEdit/GDE.App/Main/Colours/GDEColours.cs (hunk 1 above)
Comment (AltenGD) [reply GDEdit/GDE.App/Main/Colours/GDEColours.cs]: shit forgot to change that lmfao

**Pull request merged**